src/mudlet.h

```diff
@@ -361,12 +366,26 @@ class mudlet : public QMainWindow, public Ui::main_window
     void setInterfaceLanguage(const QString &languageCode);
     QList<QString> getAvailableTranslationCodes() const { return mTranslatorsMap.keys(); }
     QPair<bool, QStringList> getLines(Host* pHost, const QString& windowName, const int lineFrom, const int lineTo);
+    void setEnableFullScreenMode(const bool);
+
+    // Both of these revises the contents of the .aff file: the first will
```

Comment:
Comment needs editing - as there are now *three* methods involved and it is the third, `saveDictionary`, that replaces the contents of `.dic` with those of the supplied `QSet<QString>` (and the contents of `.aff` with details derived from that replacement).

Reply:
... in next commit.